python/mxnet/ndarray/sparse_ndarray.py

```diff
@@ -597,7 +597,7 @@ def _zeros_sparse_ndarray(stype, shape, ctx=None, dtype=None, aux_types=None, **
     ----------
     shape : int or tuple of int
         The shape of the empty array
-    stype: string
```

Comment:
for `_zeros_sparse_ndarray ` stype is not optional.. Also could you reorder the arguments in doc so that `stype` comes before `shape`?